passive/Report non static sites.js

```diff
@@ -4,58 +4,61 @@
 // Note that new passive scripts will initially be disabled
 // Right click the script in the Scripts tree and select "enable"
 
+var ScanRuleMetadata = Java.type(
+  "org.zaproxy.addon.commonlib.scanrules.ScanRuleMetadata"
+);
+
+function getMetadata() {
+  return ScanRuleMetadata.fromYaml(`
+id: 100017
+name: Non Static Site Detected
+description: >
+  A query string or form has been detected in the HTTP response body.
+  This indicates that this may not be a static site.
+solution: >
+  If this is not a static site then ignore or disable this rule.
+risk: high
```

Comment:
I understand you've recreated what was previously here but ummm imho we are safe to change this

Reply:
I guess if it is changed that should be noted in the changelog

Reply:
Should I change it to `info`?

Reply:
That seems reasonable to me

Reply:
Regarding the changelog, some names/descriptions/… were changed as well (not only in this PR), maybe add a generic entry saying that the scripts that now implement `getMetadata()` had their info revised?